package.json

```diff
@@ -38,8 +38,7 @@
     "lodash-webpack-plugin": "^0.11.4",
     "nock": "^9.0.14",
     "node-sass": "^4.5.0",
-    "react-addons-test-utils": "^15.3.1",
```

Comment:
Not required?

Reply:
Not required and deprecated since it moved into `react-dom`
https://www.npmjs.com/package/react-addons-test-utils